pkg/trace/api/api_oom_test.go

```diff
@@ -13,24 +13,24 @@ import (
 	"fmt"
 	"net/http"
 	"sync"
-	"sync/atomic"
 	"testing"
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/trace/config"
 	"github.com/DataDog/datadog-agent/pkg/trace/pb"
 	"github.com/DataDog/datadog-agent/pkg/trace/testutil"
+	"go.uber.org/atomic"
 )
 
 func TestOOMKill(t *testing.T) {
-	var kills uint64
+	var kills *atomic.Uint64 = atomic.NewUint64(0)
```

Comment:
Is the type needed? Wouldn't `kills := atomic.NewUint64(0)` work?

Reply:
Oh, good point -- that was a formulaic refactor.  I'll fix.